DESCRIPTION

```diff
@@ -31,6 +31,9 @@ Authors@R:
            email = "emwang@hbs.edu",
            role = c("aut", "cre")),
     person(given = "JMSLab",
+           role = c("cph")),
+    person(given = "Ryan",
+           family = "Kessler",
```

Comment:
Is this standard when shipping someone else's package?

Reply:
@jmshapir the reason I added this is because of the comment from [here](https://r-pkgs.org/license.html#sec-how-to-include) about how to include code you bundle:

"You also need to include some standard metadata in`Authors@R`. You should use `role = "cph"` to declare that the author is a copyright holder, with a comment describing what they’re the author of." 

Does my approach match your understanding of how we should be crediting the author? I haven't come across any packages that ship someone else's code in the same way we are, so I'm not sure what the standard is. I can look to see if I can find any packages that ship someone else's code in the same way we do.

Reply:
Thanks @nateschor and yep, I share your reading. Leaving the thread open for now per https://github.com/JMSLab/eventstudyr/pull/32#discussion_r1157894388.